app/src/wavevr/cpp/DeviceDelegateWaveVR.cpp

```diff
@@ -368,21 +398,26 @@ DeviceDelegateWaveVR::ReleaseControllerDelegate() {
 
 int32_t
 DeviceDelegateWaveVR::GetControllerModelCount() const {
-  return 1;
+  return 2;
 }
 
 const std::string
 DeviceDelegateWaveVR::GetControllerModelName(const int32_t aModelIndex) const {
   if (aModelIndex == 0) {
-    if (m.sixDoFControllerCount == 0) {
-      return "vr_controller_focus.obj";
-    }
+    return "vr_controller_focus.obj";
+  } else if (aModelIndex == 1) {
     return "vr_controller_focus_plus.obj";
   }
-
   return "";
 }
 
+// #define VRB_WAVE_EVENT_LOG_ENABLED 1
```

Comment:
nit: it seems like we don't need this comment.

Reply:
We don't strictly need it but it makes it easier to turn the event logging on and off.